libs/sdk-ui-filters/src/DateFilter/DateRangePicker/DateTimePicker.tsx

```diff
@@ -0,0 +1,164 @@
+// (C) 2022 GoodData Corporation
+import React from "react";
+import cx from "classnames";
+import { DateRangePickerInputField } from "./DateRangePickerInputField";
+import { injectIntl, WrappedComponentProps } from "react-intl";
+import { IntlWrapper } from "@gooddata/sdk-ui";
+import DayPickerInput from "react-day-picker/DayPickerInput";
+import moment from "moment";
+import { DateRangePickerInputFieldBody } from "./DateRangePickerInputFieldBody";
+import { convertPlatformDateStringToDate } from "../utils/DateConversions";
+import { TIME_FORMAT } from "../constants/Platform";
+
+interface IDateTimePickerOwnProps {
+    placeholderDate: string;
+    dateFormat: string;
+    onChange: (value: Date) => any;
+    value: Date;
+    dayPickerPropsWithDefaults: any;
+    handleDayClick: any;
```

Comment:
Please specify typings for `dayPickerPropsWithDefaults` and `handleDayClick` if possible.

Reply:
fixed